course_discovery/services.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from edx_rest_api_client.client import EdxRestApiClient
+
+
+logger = logging.getLogger()
+
+
+class CatalogApiService(object):
```

Comment:
Got it. Can you create a dev task story and label it with `quick-win` so it gets tracked by the Spartans team? Thank you